src/js/state/store.js

```diff
@@ -15,6 +15,7 @@
  */
 
 import { createStore } from 'redux';
+import withReduxEnhancer from 'addon-redux/enhancer';
```

Comment:
Why do we need to modify the app? Maybe we should keep separate store.js for storybook?

Reply:
1. One store per app convention.
2. The storybook store should has access to all our redux actions and reducers.
3. DRY. Why we should create second store, if we can use original store?  
4. This solution is consistent with documentation.